### DISCUSSION TITLE
[Feature Idea]: Custom slugs for forms

### DISCUSSION THREAD

**Discussion by jxn-30** [heyform/heyform#98] [Ideas]
I would like to be able to set a custom form ID or slug so that the resulting form URL is customized. I could then for example create a feedback form and share it under the link `{origin}/forms/feedback` which is much more memorizable then e.g. `{origin}/forms/m9Ip6nGC` and also prevents accidental typos when typing the URL.
As the form slug seems to be the form ID that could be done by allowing customization of the ID.

**Comment by iMuFeng** [comment]
The form ID must be unique, which means the `slug` also needs to be unique. If someone has already taken 'fallback', you'll need to use something like 'fallback-1.'

**Reply by jxn-30** [reply]
For smaller instances with few forms and few workspaces that should not be a problem in my eyes. "Forcig" the user to be more specific with their slug if their tried one is already used also doesm't seem to bad to me.

I see how this could possibly be frustrating on larger instances, however I don't really see this as a strong reason against that feature in general, as it is a normal "first come first serve".
